doc/samples/sid_end_device.rst

```diff
@@ -120,48 +133,36 @@ Configuration options for Sidewalk
 

```

Comment:
* ``CONFIG_SIDEWALK_SUBGHZ_SUPPORT`` -- Enables use of Sidewalk libraries with Bluetooth LE support only.

I think it should be:
* ``CONFIG_SIDEWALK_SUBGHZ_SUPPORT`` -- Enables use of Sidewalk libraries with Bluetooth LE, LoRa and FSK support. Disabling it causes to use of Sidewalk libraries with Bluetooth LE support only.